syntax/package_test.go

```diff
@@ -41,11 +41,11 @@ func TestPackageImportFromRoot(t *testing.T) {
 }
 
 func TestJsonPackageImportFromModuleRoot(t *testing.T) {
-	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `///examples/json/foo.json`)
+	AssertCodesEvalToSameValue(t, `{'location': 'Melbourne', 'name': 'foo'}`, `///examples/json/'foo.json'`)
 }
```

Comment:
The syntax much clean now, so we will only gonna support file with extension wrapped with quote?